.claude/skills/find-example/SKILL.md

```
@@ -0,0 +1,64 @@
+## Usage
+
+Run the helper with the element name (the `@MCElement(name = "...")` value):
+
+```bash
+.claude/skills/find-example/find-example.sh call
+```
+
+It lists example **directories** (with the first line of each README as a
+summary) and tutorial **files**:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)